src/lavinmq/amqp/exchange/exchange.cr

```diff
@@ -95,6 +103,7 @@ module LavinMQ
           operator_policy: @operator_policy.try &.name,
           effective_policy_definition: Policy.merge_definitions(@policy, @operator_policy),
           message_stats: current_stats_details,
+          effective_arguments: @effective_args,
```

Comment:
is that property present in rabbit too? in queue this is a comma separated string, here an array, which one do we want to use?

Reply:
> is that property present in rabbit too? 

No, this functionality does not exist in rabbit. 

>  in queue this is a comma separated string, here an array, which one do we want to use?

I started with a string and then moved on to array, but l guess I forgot to remove that `join(',')` in queue. I'll do that!